python/llm/example/CPU/Applications/autogen/README.md

```diff
@@ -0,0 +1,236 @@
+## Running AutoGen Agent Chat with BigDL-LLM on Local Models
+In this example, we use BigDL adapted FastChat to run [AutoGen](https://microsoft.github.io/autogen/) agent chat with 
+local large language models.
+
+### 1. Prepare Environment
+```bash
+# create autogen running directory
+mkdir autogen
+cd autogen
+
+# create respective conda environment
+conda create -n autogen python=3.9
+conda activate autogen
+
+# install fastchat-adapted bigdl-llm
+pip install --pre --upgrade bigdl-llm[serving]==2.5.0b20240110
```

Comment:
do we need to use this specific bigdl-llm version? maybe omit this and tell user that they should use a version > 2.5.0b20240110